src/main/java/io/vertx/ext/mongo/IndexOptions.java

```diff
@@ -16,6 +17,7 @@
   public static final boolean DEFAULT_BACKGROUND = false;
   public static final boolean DEFAULT_UNIQUE = false;
   public static final boolean DEFAULT_SPARSE = false;
+  public static final String COLLATION = "collation";
```

Comment:
We don't need this constant. The JVM will do it anyway.

Reply:
As I am using https://www.jetbrains.com/help/idea/code-inspection.html#access-inspections-and-settings I tend to refactor such things on the fly. I'll revert it and ignore my warnings then.